sidebars.ts

```diff
@@ -1960,8 +1960,163 @@ const sidebars: SidebarsConfig = {
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/set-up-scans' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Scanner Configuration',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-scanner-configuration',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/sto-techref-category' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Custom Scanning and Ingestion',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-custom-scanning-and-ingestion',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/custom-scanning' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Security Issues',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-security-issues',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/view-security-test-results' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Enforce Policies for Governance',
+          link: {
+            type: 'generated-index',
+            slug: '/category/enforce-sto-policies-for-governance',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/policies' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Remediate Issues',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-remediate-issues',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/remediations' },
+          ],
+        },
+        'security-testing-orchestration/jira-integrations',
+        {
+          type: 'category',
+          label: 'Exempt Issues',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-exempt-issues',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/exemptions' },
+          ],
+        },
+        {
+          type: 'category',
+          label: 'Set Notifications',
+          link: {
+            type: 'generated-index',
+            slug: '/category/sto-set-notifications',
+          },
+          collapsed: true,
+          items: [
+            { type: 'autogenerated', dirName: 'security-testing-orchestration/notifications' },
+          ],
+        },
+        {
+          type: "html",
+          value: "Resources & Troubleshooting",
```

Comment:
```suggestion
          value: "Troubleshooting & Resources",
```

For consistency with other modules.